test-assignments/frontend-react-assignment/src/hooks/redux.ts

```diff
@@ -1,7 +1,6 @@
 import { useDispatch, useSelector } from 'react-redux'
-import type { TypedUseSelectorHook } from 'react-redux'
-import type { RootState, AppDispatch } from '../store/store'
+import type { AppDispatch, RootState } from '../store/store'
 
 // Use throughout your app instead of plain `useDispatch` and `useSelector`
-export const useAppDispatch: () => AppDispatch = useDispatch
-export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector
+export const useAppDispatch = useDispatch.withTypes<AppDispatch>()
```

Comment:
update to latest guides
https://react-redux.js.org/using-react-redux/usage-with-typescript#define-typed-hooks